api/app/tests/apps/api/v1/test_private_signal_endpoint.py

```diff
@@ -215,9 +218,9 @@ def test_history_action_filters(self):
 
     # -- write tests --
 
-    @patch("signals.apps.api.v1.validation.AddressValidation.validate_address_dict",
+    @patch("signals.apps.api.v1.validation.address.base.BaseAddressValidation.validate_address",
            side_effect=AddressValidationUnavailableException)  # Skip address validation
-    def test_create_initial(self, validate_address_dict):
+    def test_create_initial(self, validate_address):
```

Comment:
Add a check that for the `bag_validated` boolean. I the model we set that field to `False` if we cannot validate, but I think we should check it explicitly in the tests (so that we catch regressions when we change the implementation).

Reply:
I am rather not fixing tests that where already in the test-suite. The only thing that changed here is the name and place of the patched model/function.

Reply:
ok